packages/11ty/_plugins/figures/iiif/config.js

```diff
@@ -109,6 +109,15 @@ module.exports = (eleventyConfig) => {
         resize: {
           width: 800
         }
+      },
+      /**
+       * Transformation applied to IIIF resources for use in inline figures
+       */
+      {
+        name: 'static-inline-figure',
+        resize: {
+          width: 626
+        }
```

Comment:
I guess we don't actually need this one since `800px` is close enough...

Reply:
@anderspollack Actually, we should bump the print-image size up to give us images that will print at 300ppi if they're shown at our default full page size, and also add the `withoutEnlargement` attribute to keep smaller images from being enlarged. I did this on _French Silver_.

```
resize: {
  width: 2025,
  withoutEnlargement: true
}
````